src/EventExport/Format/JSONLD/JSONLDEventFormatter.php

```diff
@@ -104,17 +115,32 @@ function ($term) use ($includedTerms) {
                 $eventObject->terms = array_values($filteredTerms);
             }
 
+            if (isset($this->calendarSummaryRepository) && in_array('calendarSummary', $includedProperties)) {
+                $urlParts = explode('/', $eventObject->{'@id'});
+                $eventId = array_pop($urlParts);
+                $eventObject->calendarSummary = $this->addCalendarSummary($eventId);
+            }
+
             // filter out base properties
             foreach ($eventObject as $propertyName => $value) {
                 if (!in_array($propertyName, $includedProperties)) {
                     unset($eventObject->{$propertyName});
                 }
             }
 
-
             $event = json_encode($eventObject);
         }
 
         return $event;
     }
+
+    private function addCalendarSummary(string $eventId): string
+    {
+        try {
+            $calendarSummary = $this->calendarSummaryRepository->get($eventId, ContentType::plain(), Format::sm());
+        } catch (SummaryUnavailableException $exception) {
+            $calendarSummary = $exception->getMessage();
```

Comment:
I would leave the summary empty instead of putting the exception message inside the exported JSON.